docs/sample_config.yaml

```diff
@@ -643,6 +643,11 @@ uploads_path: "DATADIR/uploads"
 #
 #enable_registration: false
 
+# Optional time duration (in weeks) after which a new account is denied
+# access. 0 means no expiration.
+#
+#validity_period: 0
```

Comment:
There's a `parse_duration` that will let you specify things like `5w` or `2d` etc, which might make sense to use?

Reply:
I'd also use `null` to mean no expiry to be honest, rather than magic value. 

I think we're also moving to the style where the config option has an example value rather than the default, so:

```yaml
# Optional duration after which a new account must be re-validated
# or denied access. This is disabled by default.
#
#validity_period: 6w
```

Reply:
In fact, we're probably going to add more options here? So might be worth making it a dict up front?